packages/react/src/components/VideoPlayer/__stories__/VideoPlayer.stories.js

```diff
@@ -51,3 +45,41 @@ export const Default = ({ parameters }) => {
     </div>
   );
 };
+
+export const aspectRatio1x1 = ({ parameters }) => {
+  return <Default parameters={parameters} />;
+};
+
+aspectRatio1x1.story = {
+  name: 'Aspect ratio 1:1',
+  parameters: {
+    knobs: {
+      VideoPlayer: () => {
+        return {
+          showCaption: true,
+          aspectRatio: '1x1',
+          videoId: '1_9h94wo6b',
+        };
+      },
+    },
+  },
+};
+
+export const aspectRatio4x3 = ({ parameters }) => (
+  <Default parameters={parameters} />
+);
+
+aspectRatio4x3.story = {
+  name: 'Aspect ratio 4:3',
+  parameters: {
+    knobs: {
```

Comment:
Given there is no parameter controlled by a knob here:

```suggestion
    props: {
```